python/ez_setup.py

```diff
@@ -1,284 +1,332 @@
-#!python
-
-# This file was obtained from:
-#   http://peak.telecommunity.com/dist/ez_setup.py
-# on 2011/1/21.
-
+#!/usr/bin/env python
```

Comment:
Actually, it would be better to just rip out `ez_setup.py` altogether:  modern Python packages can just assume the user can get `setuptools` installed without it.